docs/en/advanced-concepts/settings.md

```diff
@@ -52,25 +52,30 @@ Settings are implemented via the following API endpoints:
-|`enableMultiWindows`| If enabled then the page source and xpath lookup will include all windows available to the accessibility rather than the active one only. It might help to resolve finding elements on windows like `android.widget.PopupWindow`. Defaults to `false` and returns an active root window. Read [appium-uiautomator2-server#301](https://github.com/appium/appium-uiautomator2-server/pull/301) for mroe details. |`true` or `false` |
-
+| `actionAcknowledgmentTimeout` | Same as: [setActionAcknowledgmentTimeout](https://developer.android.com/reference/android/support/test/uiautomator/Configurator.html#setActionAcknowledgmentTimeout(long)). If a negative value is given, it would set to default (3 * 1000 milliseconds). Handled by [UiAutomator Configurator](https://developer.android.com/reference/android/support/test/uiautomator/Configurator.html) in Android API 18 and above. | e.g., `5000` |
+| `allowInvisibleElements` |  Controls whether the Android device should show all elements, visible and invisible. Defaults to `false`. | `false` or `true` |
+| `enableMultiWindows` |  If enabled then the page source and xpath lookup will include all windows available to the accessibility rather than the active one only. It might help to resolve finding elements on windows like `android.widget.PopupWindow`. Defaults to `false` and returns an active root window. Read [appium-uiautomator2-server#301](https://github.com/appium/appium-uiautomator2-server/pull/301) for more details. | `true` or `false` |
+| `enableNotificationListener` |  Controls whether the Android device should enable or disable the `NotificationListener`. Defaults to `true`.  | `false` or `true` |
+| `keyInjectionDelay` | Same as: [setKeyInjectionDelay](https://developer.android.com/reference/android/support/test/uiautomator/Configurator.html#setKeyInjectionDelay(long)). If a negative value is given, it would set to default(0 milliseconds). Handled by [UiAutomator Configurator](https://developer.android.com/reference/android/support/test/uiautomator/Configurator.html) in Android API 18 and above. | e.g., `5000` |
+| `mjpegBilinearFiltering` | Controls whether or not to apply bilinear filtering to source screenshot resize algorithm. Enabling this flag may improve the quality of the resulting scaled bitmap, but will introduce a [small] performance hit. Defaults to `false`. | `false` or `true` |
+| `mjpegScalingFactor` | Controls the scaling factor of streamed screenshots. 100 means no scaling is applied (e.g. 100% of the original size). The bigger image size is the more CPU performance is needed to encode it. Defaults to `50`. | `1` to `100` |
```

Comment:
...encode it. Integer between `1` and 100 are available. Defaults to `50`

(Addresses available type of number)
mjpegServerFramerate is as well.

Reply:
Busy morning. I will update ASAP. Thanks for the review @KazuCocoa

Reply:
This is resolved w/ the latest update(s)